tool_commands/raster_checker/raster_checker_log.py

```diff
@@ -356,7 +360,7 @@ def write_log(self, entries_metadata):
         timestr = time.strftime("_%Y%m%d_%H%M")
         log_dir, sqltname_with_ext = os.path.split(self.sqlite_path)
         sqltname_without_ext = os.path.splitext(sqltname_with_ext)[0]
-        self.log_path = log_dir + "/" + sqltname_without_ext + timestr + ".logger"
+        self.log_path = log_dir + "/" + sqltname_without_ext + timestr + ".log"
```

Comment:
Gaat dit goed op windows?
Het is beter om [os.path.join](https://docs.python.org/3.7/library/os.path.html#os.path.join) of [pathlib](https://docs.python.org/3.5/library/pathlib.html#basic-use) te gebruiken.

os.path.join:
`self.log_path = os.path.join(log_dir, sqltname_without_ext + timestr + ".log")`

pathlib:
`self.log_path = log_dir / sqltname_without_ext + timestr + ".log"`